### PR TITLE
Add documentation for proxy pass through flag

### PR DESCRIPTION
**Pull request by kapishmalik**
@tomakehurst Added documentation for proxy pass-through flag feature. Needs a review.

### CODE REVIEW

File: _docs/proxying.md [hunk 1]
```diff
@@ -158,6 +158,8 @@ stubFor(get(urlEqualTo("/users/12345.json"))
   .withStatus(503)));
 ```
 
+Also, we can enable/disable pass through unmatched requests to the target indicated by the original requests by enabling/disabling proxyPassThrough flag.  This flag can be set either by passing option while running standalone jar or by updating global settings through API.
```
Comment (tomakehurst) [inline _docs/proxying.md:161]: Please can we have a link to the CLI parameter here?

And also an example in both Java and JSON of how to set the flag via a global call?

Comment (kapishmalik) [reply _docs/proxying.md]: sure, incorporated

File: _docs/proxying.md [hunk 2]
```diff
@@ -158,6 +158,21 @@ stubFor(get(urlEqualTo("/users/12345.json"))
   .withStatus(503)));
 ```
 
+Also, we can enable/disable pass through unmatched requests to the target indicated by the original requests by enabling/disabling proxyPassThrough flag.
+By default, flag is set to true. This flag can be enabled/disabled either by passing CLI option while running jar as described in [Running Standalone](/docs/running-standalone/#command-line-options) or by updating global settings.
+
+Update proxyPassThrough flag via Java client
+```java
+WireMockServer wireMockServer = new WireMockServer(options().proxyPassThrough(false));
+```
```
Comment (tomakehurst) [inline _docs/proxying.md:167]: Can we also show how to do this at runtime in Java i.e. by updating the `GlobalSettings`?

Comment (tomakehurst) [reply _docs/proxying.md:167]: And explain that the `options()` approach is for startup, where as the `GlobalSettings` allows it to be changed without a restart.

Comment (kapishmalik) [reply _docs/proxying.md:167]: incorporated

**Comment by tomakehurst** [comment]
Looks good, thanks!

**Pull request merged**